### DISCUSSION TITLE
Can I move/shift-right a range of cells in a column?

### DISCUSSION THREAD

**Discussion by osevill** [saulpw/visidata#1935] [Q&A] [answered]
My scenario is the csv file I'm working on has data integrity issues, where some rows have fewer columns than most other rows, e.g., the header row and 99% of the remaining rows have 10 columns, but 1% have only 8 columns, where the data after these missing columns is erroneously shifted left two columns.
Can I use visidata to "push" a range of cells some number of columns to the right, inserting blank cells, in order to correct the original problem?
My assumption is that I can first sort the original file in visidata by the last header column, so that the rows missing a value in that column will sort together to the top of the document.
Thanks.

**Comment by saulpw** [comment]
Hi @osevill, there's an experimental command "slide-cells" in the develop branch.  Add this to your visidatarc:
```
import experimental.slide_cells
```

Then try using the slide-cells commands.  You may have to do some exploration or tweaking to get exactly the kind of command you want.

**Reply by osevill** [reply]
Thanks!  I updated to the dev branch by entering this in the MacOS terminal:
pip3 install git+https://github.com/saulpw/visidata.git

and now I'm on VisiData v2.12dev.

I created a .visidatarc file in my home directory and added just this one line:
import experimental.slide_cells

...but when I launch vd, I see the following at the bottom of the window:
ModuleNotFoundError: No module named 'experimental'

Is slide_cells a plugin? and if so, where can I download it?

Thank you.

**Reply by anjakefala** [reply, marked as answer]
Saul was missing an element! It is `import visidata.experimental.slide_cells`.

**Reply by saulpw** [reply]
Oh, thank you, sorry about that!  Yes you need the `visidata.` since it's packaged with the visidata module.

**Reply by osevill** [reply]
Thanks both!  Updated the import statement, and it worked.
